src/libs/actions/Download.js

```diff
@@ -0,0 +1,18 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * Immediate indication whether the an attachment is being downloaded.
+ *
+ * @param {String} sourceID
+ * @param {Boolean} isDownloading
+ * @returns {Promise}
+ */
+function setDownload(sourceID, isDownloading) {
+    return Onyx.merge(`${ONYXKEYS.COLLECTION.DOWNLOAD}${sourceID}`, {isDownloading});
+}
+
+export {
+    // eslint-disable-next-line import/prefer-default-export
```

Comment:
If we move `clearDownloads` to this file we can also remove the lint suppression
```suggestion

```

Reply:
Done